lib/mongo_client.js

```diff
@@ -118,6 +118,17 @@ const closeOperation = require('./operations/mongo_client_ops').closeOperation;
  * @param {number} [options.minSize] If present, the connection pool will be initialized with minSize connections, and will never dip below minSize connections
  * @param {boolean} [options.useNewUrlParser=false] Determines whether or not to use the new url parser. Enables the new, spec-compliant, url parser shipped in the core driver. This url parser fixes a number of problems with the original parser, and aims to outright replace that parser in the near future.
  * @param {boolean} [options.useUnifiedTopology] Enables the new unified topology layer
+ * @param {object} [options.autoEncryption] Optionally enable client side auto encryption
```

Comment:
request: let's make this its own `@typedef`. That way we can also update the documentation for `MongoClient.connect` easily

Reply:
I started down that path, but didn't we prove that it doesn't show up correctly in our current documentation?

Reply:
How to write a typedef: https://devdocs.io/jsdoc/tags-typedef

Example in our code: 
+ https://github.com/mongodb/node-mongodb-native/blob/next/lib/core/transactions.js#L68
+ https://github.com/mongodb/node-mongodb-native/blob/next/lib/core/sessions.js#L278

What it looks like:
+ http://mongodb.github.io/node-mongodb-native/3.2/api/ClientSession.html#withTransaction
+ http://mongodb.github.io/node-mongodb-native/3.2/api/global.html#TransactionOptions